core/modules/views_ui/views_ui.admin.inc

```diff
@@ -442,8 +442,17 @@ function views_ui_add_form($form, &$form_state) {
     '#validate' => array('views_ui_wizard_form_validate'),
     '#submit' => array('views_ui_add_form_save_submit'),
   );
+  
+  $path = 'admin/structure/views';
+  $options = backdrop_parse_url($path);
```

Comment:
Let's just remove this line. With a hard-coded path like 'admin/structure/views', it won't ever create any `$options` that we care about (e.g. a `query` value).